lib/std/special/compiler_rt/clear_cache.zig

```diff
@@ -142,3 +149,10 @@ pub fn clear_cache(start: usize, end: usize) callconv(.C) void {
 
 // Darwin-only
 extern fn sys_icache_invalidate(start: usize, len: usize) void;
+// BSD-only
+const arm_sync_icache_args = extern struct {
```

Comment:
Why do these end up here instead of `std/os/bits`?

Reply:
I've followed what @andrewrk did for `sys_icache_invalidate`, we can reorganize the bits whenever we decide where to put those.